src/pages/HomePage/hooks.ts

```diff
@@ -19,22 +19,27 @@ import type {
   UserActivityType,
 } from "@/services/copper/types";
 
-type UseContact = () => {
+type UseContact = (activitiesPage: number) => {
   isLoading: boolean;
   contact: Contact;
   account: Account;
   activities: Activity[];
   notes: Activity[];
+  isLoadingActivities: boolean;
   opportunities: Opportunity[];
   activityTypes: UserActivityType[];
 };
 
-const useContact: UseContact = () => {
+const useContact: UseContact = (activitiesPage) => {
+  const [userActivityTypes, setUserActivityTypes] = useState<UserActivityType[]>([]);
+  const [noteTypes, setNoteTypes] = useState<UserActivityType[]>([]);
+  const [activities, setActivities] = useState<Activity[]>([]);
+
   const account = useQueryWithClient([QueryKey.ACCOUNT], getAccountService);
 
   const contact = useLinkedContact();
 
-  const contactId = useMemo(() => get(contact, ["contact", "id"]), [contact]);
+  const contactId = useMemo(() => contact?.contact?.id, [contact]);
```

Comment:
No need for `memo` here as the operation is simple.

Reply:
done